config/locales/en.yml

```diff
@@ -456,9 +456,9 @@ en:
             will_need_chromebooks:
               blank: Tell us whether the school will need Chromebooks
             school_or_rb_domain:
-              blank: Enter a domain in the right format, like myschool.org.uk
-              invalid: Enter a domain in the right format, like myschool.org.uk
-              invalid_domain: Enter a domain registered for G Suite for Education
+              blank: Enter an email domain registered for G Suite for Education
+              invalid: Enter an email domain registered for G Suite for Education
```

Comment:
I don't like that we're removing the examples here. Why are we not keeping the "in the right format" message?

Reply:
Question for @ralph-hawkins ?

Reply:
I've added the ', like myschool.org.uk' back in to the error messages, what do you think now?

Reply:
Yeah — sorry that wasn't my intention to change all the error messages — they're better kept specific.